templates/core/terraform/state-store/statestore.tf

```diff
@@ -25,3 +25,34 @@ resource "azurerm_cosmosdb_sql_database" "tre-db" {
   account_name        = azurerm_cosmosdb_account.tre-db-account.name
   throughput          = 400
 }
+
+resource "azurerm_private_dns_zone" "statestore" {
+  name                = "privatelink.statestore.azure.net"
```

Comment:
This should be `privatelink.documents.azure.com.`.

Reply:
I can change it. How did you come up with the name though? Is this documented?

Reply:
https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-dns

There are a few for Cosmos - not sure which is the one we need.

Reply:
Thanks½